ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -1868,6 +1868,21 @@ management_api_gnmi:
   octa:
 ```
 
+#### Management API Models
+
+```yaml
+management_api_models:
+  providers:
+    - name: < provider1 (sysdb | smash) >
```

Comment:
```suggestion
    - name: < sysdb | smash >
```

Reply:
These are list of available providers:

```
  aft            Configure AFT provider
  http-commands  Configure Http Commands provider
  ipfix          Configure IPFIX provider
  sflow          Configure sFlow provider
  smash          Configure Smash provider
  sysdb          Configure Sysdb provider
```

Reply:
So we can simply say:

```- name: < provider_name >```

or we can provide the list of options as below:
- name: < aft | http-commands | ipfix | sflow | sysdb | smash >

Reply:
Since we need to implement the specific data models differently, we should either define it as a dict with static keys for the supported data models, or as a list with `name` key, where we show the supported values.

Reply:
so currently since we are implementing only `smash` and `sysdb` should we keep it as follows to avoid any confusion for currently supported providers.
```shell
- name: < sysdb | smash >
```